Userland/Libraries/LibAudio/FlacLoader.cpp

```diff
@@ -966,13 +966,20 @@ ErrorOr<u64> read_utf8_char(BigEndianInputBitStream& input)
     } else if ((start_byte & 0b11000000) == 0b10000000) {
         return Error::from_string_literal("Illegal continuation byte");
     }
-    // This algorithm is too good and supports the theoretical max 0xFF start byte
+    // This algorithm is too good and supports the theoretical max 0xFF start byte.
```

Comment:
What's with the "too good" here and "This specialized UTF-8 decoder is more powerful than a normal UTF-8
decoder anyways" in the commit message? Please make quantitative statements instead.